File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/04_For_Loop_Exercise/05_Salary.js
function salary(input) {
    let penaltyFacebook = 150;
    let penaltyInstagram = 100;
    let penaltyReddit = 50;

    let openedTabs = Number(input[0]);  // 10;
    let salary = Number(input[1]);

    // console.log(input.length);
    // for (let i = 0; i < openedTabs; i++) {
    for (let i = 0; i < input.length - 2; i++) {
        let websiteName = input[i + 2];  // 1. iter => input[0 + 2] == input[2];
        // console.log(websiteName);
        if (websiteName == 'Facebook') {
            salary -= penaltyFacebook;
        } else if (websiteName == 'Instagram') {
            salary -= penaltyInstagram;
        } else if (websiteName == 'Reddit') {
            salary -= penaltyReddit;
        }
    }

    if (salary <= 0) {
        console.log('You have lost your salary.');
    } else {
        console.log(salary);
    }
}


// salary([
//     "10",
//     "750",
//     "Facebook",
//     "Dev.bg",
//     "Instagram",
//     "Facebook",
//     "Reddit",
//     "Facebook",
//     "Facebook",
// ]);

// salary([
//     "3",
//     "500",
//     "Github.com",
//     "Stackoverflow.com",
//     "softuni.bg",
// ]);

File: Python_Courses/Python_Basics/Python_PB_2024_10/04_For_Loop_Exercise/05_Salary.py
# •	"Facebook" -> 150 лв.
# •	"Instagram" -> 100 лв.
# •	"Reddit" -> 50 лв.

opened_tabs = int(input())  # 4
salary = float(input())

for _ in range(opened_tabs):  # for tab in range(4):  [0, 1, 2, 3]
    current_tab = input()  # 'Facebook', ...
    if current_tab == 'Facebook':
        salary -= 150
    elif current_tab == 'Instagram':
        salary -= 100
    elif current_tab == 'Reddit':
        salary -= 50

    if salary <= 0:
        break

if salary <= 0:
    print('You have lost your salary.')
else:
    print(int(salary))
